src/security/credentials.rs

```diff
@@ -5,7 +5,71 @@ use std::ptr;
 
 use crate::error::{Error, Result};
 use crate::grpc_sys::{self, grpc_channel_credentials, grpc_server_credentials};
-use libc::c_char;
+
+#[repr(u32)]
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
+pub enum CertificateRequestType {
+    /// Server does not request client certificate.
+    ///
+    /// The certificate presented by the client is not checked by the server at
+    /// all. (A client may present a self signed or signed certificate or not
+    /// present a certificate at all and any of those option would be accepted)
+    DontRequestClientCertificate = 0,
```

Comment:
```suggestion
    DontRequestClientCertificate = grpcio_sys::grpc_ssl_client_certificate_request_type::GRPC_SSL_DONT_REQUEST_CLIENT_CERTIFICATE,
```

How about using `grpcio_sys::grpc_ssl_client_certificate_request_type::GRPC_SSL_DONT_REQUEST_CLIENT_CERTIFICATE`?